large-message-core/src/main/java/com/bakdata/kafka/AbstractLargeMessageConfig.java

```diff
@@ -290,10 +309,13 @@ private boolean enableAmazonS3PathStyleAccess() {
                 return Optional.of(oidcProvider);
             }
 
-
         }
 
         return Optional.empty();
     }
 
+    private static BlobStorageClient createGoogleStorageClient() {
+        final Storage storage = StorageOptions.newBuilder().setProjectId(GOOGLE_CLOUD_PROJECT_ID).build().getService();
```

Comment:
This doesn't use the user supplied project id, but the config name itself, i.e. `large.message.gs.project.id`.

Reply:
```suggestion
        final Storage storage = StorageOptions.newBuilder().setProjectId(this.getString(GOOGLE_CLOUD_PROJECT_ID)).build().getService();
```